test/files/run/print-args.scala

```diff
@@ -26,6 +27,7 @@ object Test extends DirectTest {
     """
   def show() = {
     assert(!compile())
+    //assertEquals(expected.linesIterator.toList, Files.readAllLines(argfile).asScala.toList)
```

Comment:
debug cruft. The `expected` needs `.trim` and assert could be
```
    import scala.tools.testkit.AssertUtil.assertSameElements
    assertSameElements(expected.linesIterator, Files.readAllLines(argfile).asScala)
```
The test is because `-Vprint-args` was broken; the `-Xsource` is not relevant.

I'll submit the tweak separately. :)